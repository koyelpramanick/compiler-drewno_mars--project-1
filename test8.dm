"testing all \t of these \n inside a \" string   "
890823488934201
"can you escape a \k?"
"illegal char inside string &*^^&*(("
1+2-(87*5)/2
"here's an illegal char: " &
"let's not finish this string