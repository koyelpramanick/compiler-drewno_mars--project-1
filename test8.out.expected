STRINGLITERAL:"testing all \t of these \n inside a \" string   " [1,1]
INTLITERAL:0 [2,1]
STRINGLITERAL:"illegal char inside string &*^^&*((" [4,1]
INTLITERAL:1 [5,1]
CROSS [5,2]
INTLITERAL:2 [5,3]
DASH [5,4]
LPAREN [5,5]
INTLITERAL:87 [5,6]
STAR [5,8]
INTLITERAL:5 [5,9]
RPAREN [5,10]
SLASH [5,11]
INTLITERAL:2 [5,12]
STRINGLITERAL:"here's an illegal char: " [6,1]
EOF [7,30]
